### DISCUSSION TITLE
stable replays exported with relax don't have working relax

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#19311] [Q&A] [answered]
https://user-images.githubusercontent.com/78441708/180596598-bcd5471c-61c5-412f-ac69-e4cf4123e751.mp4

this is a replay exported from stable which uses relax, when imported into lazer relax no longer functions. 

map & replay
[files.zip](https://github.com/ppy/osu/files/9173357/files.zip)
NOTE: this map has a modified `.osu`, for some reason this won't work in stable. its not related to this issue (im 99% sure) but it **should** work when you import to lazer. this is not necessarily supposed to work on stable. it is not an issue as far as im aware, because we modified it.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/9243

**Reply by Axilotl17** [reply]
sorry, my bad.
